core/Db/Schema/Mysql.php

```diff
@@ -515,11 +515,12 @@ public function createTable($nameWithoutPrefix, $createDefinition)
         $dbSettings   = new Db\Settings();
         $charset      = $dbSettings->getUsedCharset();
 
-        $statement = sprintf("CREATE TABLE IF NOT EXISTS `%s` ( %s ) ENGINE=%s DEFAULT CHARSET=%s ;",
+        $statement = rtrim(sprintf("CREATE TABLE IF NOT EXISTS `%s` ( %s ) ENGINE=%s DEFAULT CHARSET=%s %s;",
```

Comment:
guess the `rtrim` is actually obsolete, as the query always ends with a `;`

Reply:
@sgiehl Thanks for pointing that out. :)